scripts/superdesk-authoring/authoring.js

```diff
@@ -1502,7 +1505,7 @@
                 };
 
                 $scope.save_enabled = function() {
-                    return $scope.dirty || $scope.item._autosave;
+                    return (confirm.dirty = $scope.dirty || $scope.item._autosave);
```

Comment:
@petrjasek , slightly changed it as interested only in scope.dirty() or keep both?